azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/artifacthandler/FTPArtifactHandlerImpl.java

```diff
@@ -45,6 +45,8 @@ protected boolean isResourcesPreparationRequired(final DeployTarget target) {
 
     @Override
     public void publish(final DeployTarget target) throws IOException, MojoExecutionException {
+        this.mojo.getLog().info(String.format(DEPLOY_START, target.getType(), target.getName()));
```

Comment:
`Deploying Web App to xxx`? this log looks strange.

Reply:
Typo, should be Deploying to Web App xxx, to be different from Deploying to Deployment Slot xxx.